src/test/java/com/cloudbees/hudson/plugins/folder/computed/ComputedFolderTest.java

```diff
@@ -443,6 +443,16 @@ public void triggersRoundtrip() throws Exception {
 
     }
 
+    @Test
+    public void cleanTriggers() throws Exception {
+        SampleComputedFolder s = r.jenkins.createProject(SampleComputedFolder.class, "s");
+        s.addTrigger(new PeriodicFolderTrigger("30m"));
```

Comment:
nit: Just to complete the circle, so in the same test we check that the trigger is added and removed 

```suggestion
        s.addTrigger(new PeriodicFolderTrigger("30m"));
        
        assertEquals(1, s.getTriggers().size());
```